pof/src/types.rs

```diff
@@ -1625,6 +1625,16 @@ impl Model {
         (out_vec, out_idx)
     }
 
+    pub fn do_for_recursive_subobj_children(&self, id: ObjectId, f: &mut impl FnMut(&SubObject)) {
```

Comment:
```suggestion
    pub fn do_for_recursive_subobj_children<'a>(&'a self, id: ObjectId, f: &mut impl FnMut(&'a SubObject)) {
```
This shouldn't change anything but it's a slightly more accurate function signature, it says that the function `f` is being called on subobjects that derive from `self`.